Library/Formula/bzr-rewrite.rb

```diff
@@ -0,0 +1,63 @@
+class BzrRewrite < Formula
+  desc "Bazaar plugin to support rewriting revisions and rebasing"
+  homepage "https://launchpad.net/bzr-rewrite"
+  url "https://launchpad.net/bzr-rewrite/trunk/0.6.3/+download/bzr-rewrite-0.6.3.tar.gz"
+  sha256 "f4d0032a41a549a0bc3ac4248cd4599da859174ea33e56befcb095dd2c930794"
+
+  bottle :unneeded
+  depends_on "bazaar"
```

Comment:
Since there's other changes to be made, an empty line between the `bottle :xyz` and `depends_on` would be great as well, Thanks!